### DISCUSSION TITLE
Kit App (on mac Intel) won't open (even after fresh re-install)

### DISCUSSION THREAD

**Discussion by mtin79** [johnlindquist/kit#726] [Errors] [answered]
Hi there,
I am using an intel based macbook (2019), Mac os x version 12.2.
Just downloaded the Intel Macversion of the Kit App on the Website.
Saw the welcome dialog once and the write and run scripts window afterwards.
Then i closed it and first i could see the app icon open and being available.
But no window any more.

De-installed and reinstalled and now the App is not even opening any longer.

Any ideas?

**Comment by mtin79** [comment, marked as answer]
So to answer my question and why the initiall default shortcut doesn't work for parts of the world who don't have an US Keyboard Layout.
After installing ScriptKit Application there seems to be no Application Icon in the Mac Os X Dock.
After it's installed ScriptKit lives in the menubar and maybe hidden at first if you use a MenuBar extension like Bartender.
The short should be initially set to something else than ⌘+; because my german keyboard layout wasn't able to invoke it.

**Reply by johnlindquist** [reply]
Do you have any ideas why a German keyboard can't invoke <kbd>cmd+;</kbd>?  I'm not familiar enough with the layout to understand the problem.

**Reply by mtin79** [reply]
Hi John, this is due to that the ";" character is only accessible when you keep the "⇧" - key pressed.
So in case you create a shortcut with "⇧" - key the ","- key is used instead. ";" is on the same key as the "," key.

<img width="447" alt="Screenshot 2022-02-10 at 16 23 41" src="https://user-images.githubusercontent.com/46390/153439090-83b9ad28-9219-4c89-94b0-a8913fdd5494.png">

**Reply by johnlindquist** [reply]
Thank you so much for the follow-up. We're well past the point that we can change the default shortcut. 

I'll have to see if there's a way to detect the keyboard layout and offer alternates based on the keyboard. 🤔
